### DISCUSSION TITLE
New to github - I need some help

### DISCUSSION THREAD

**Discussion by mornaistar** [community/community#22293] [New to GitHub]
<p>Hi there!</p>
<p>I’m new to Github (200% new) and i need some help.</p>
<p>I’m working on a project,</p>
<p>Right now i’m on development stage but in the near future i will be deploying.</p>
<p>I decided to create a repo to help me with version control and that way i can have a [<strong>development /</strong>** production**] system.</p>
<p>So i created a repo &lt;myproject&gt; with a master branch and a dev branch.</p>
<p>So far so good. </p>
<p>My question is:</p>
<p>In the future my code must be running from a server (running xampp).</p>
<p>What’s the easiest way to get my master branch on the server?</p>
<p>Is it possible to auto sync from master branch?</p>
<p>Do i have to access the server and pull the data?</p>
<p>What’s the easiest (noob-friendly) way to get my master branch on the server?</p>
<p>Thanks for all the help and sorry for my English</p>
<p>Best regards,</p>
<p>Artur Mendes</p>

**Comment by Wabri** [comment, marked as answer]
<p>Hi <a class="mention" href="/mornaistar">@mornaistar</a>,</p>
<p>Welcome to Github Community Forum!</p>
<p>I dont know if it’s difficult to use for a beginner but i think you can use this gist: <a href="https://gist.github.com/oodavid/1809044" rel="nofollow noopener">gist.github.com/oodavid/1809044</a></p>
<p>-Gabriele-</p>
<p>Mark helpful posts with Accept as Solution to help other users locate important info. Don’t forget to give Kudos for great contents!</p>

**Comment by mpboom** [comment]
<p>Hi <a class="mention" href="/mornaistar">@mornaistar</a> ,</p>
<p>Did the snippet from <a class="mention" href="/wabri">@wabri</a> help you out or do you need further assistance?</p>

**Comment by mornaistar** [comment]
<p>Thanks a lot! :slight_smile:</p>
